db/migrate/20220924224924_add_description_to_legislation_question_translations.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToLegislationQuestionTranslations < ActiveRecord::Migration[6.0]
+  def change
+    add_column :legislation_question_translations, :description, :string
```

Comment:
I think we usually use `text` for descriptions :thinking:.

```suggestion
    add_column :legislation_question_translations, :description, :text
```